### DISCUSSION TITLE
Object Category Definition Samples

### DISCUSSION THREAD

**Discussion by vinukumar-vs** [Sunbird-Knowlg/Community#104] [Documentation Issues]
I was creating the new primary object definition. While going through the documentation, I have noticed the payload of the the below API is empty for 

{{baseUrl}}/object/category/definition/v1/create

https://www.postman.com/sunbird-building-blocks/workspace/sunbird-knowlg/request/25463377-e1630809-4f91-4414-a9df-f9b3a9ffefe9

> "objectMetadata": {
        "config": {},
        "schema": {}
      }

Also in the above API documentation, I didn't notices "forms" also can be configured. 

Adding some examples of request payload with different samples will  help to create new object category definition(OCD). 
If you can add the samples as "Add example" in the same postman api request will help.

@Krishnaj20

**Comment by pallakartheekreddy** [comment]
@vinukumar-vs Thanks for bringing this. We will improve our documentation and will update the link here once we finished.
